apiserver/allfacades.go

```diff
@@ -166,8 +166,9 @@ func AllFacades() *facade.Registry {
 	reg("LogForwarding", 1, logfwd.NewFacade)
 	reg("MachineActions", 1, machineactions.NewExternalFacade)
 
-	reg("MachineManager", 2, machinemanager.NewFacade)
-	reg("MachineManager", 3, machinemanager.NewFacade) // Version 3 adds DestroyMachine and ForceDestroyMachine.
+	reg("MachineManager", 2, machinemanager.NewFacadeV3)
```

Comment:
This is a little confusing. Perhaps we leave v2 and v3 as NewFacade and make v4 NewFacadeV4